db/students.ts

```diff
@@ -0,0 +1,23 @@
+import { studentsTable } from './tables';
+import db from './db';
+
+const insert = async (email: string): Promise<void> => {
+  try {
+    const existingMail = await db
+      .from(studentsTable)
+      .where(`${studentsTable}.email`, email);
+
+    if (existingMail.length === 0) {
+      await db(studentsTable).insert({
+        email,
+      });
+    }
```

Comment:
I think you can use something like `insert({ email }).onConflict("email").ignore();` to insert only if it doesn't exist.